### PR TITLE
DOC Elemental versioned badge

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/developer-docs/issues/474

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Include a screenshot - if I didn't know what work had been done I'd assume this was talking about the dots that already exist.

**Comment by emteknetnz** [comment]
Updated

File: en/08_Changelogs/5.2.0.md [hunk 1]
```diff
@@ -215,6 +216,14 @@ SilverStripe\ErrorPage\ErrorPage:
     - 500
 ```
 
+### Versioned status badges for elemental {#elemental-badges}
+
+Elemental content blocks now feature versioned status badges which adds a textual indicator of whether the block is in draft and/or a modified state. This textual indicator is in addition to the dot icon indicator that was already there. This functionality is only enabled if the [`Versioned`](api:SilverStripe\Versioned\Versioned) extension is enabled which it is by default.
+
+![elemental-versioned-badge](../_images/changeslogs/5.2.0/elemental-badge.png)
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.2.0.md]: Path is wrong, doesn't render. I think this is all that's needed but double check before just copying this:
```suggestion
![elemental-versioned-badge](../_images/changelogs/5.2.0/elemental-badge.png)
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/5.2.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/08_Changelogs/5.2.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM.

Won't be in beta changelog but that's not a big deal.

File: en/08_Changelogs/5.2.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/08_Changelogs/5.2.0.md]: Updated

Comment (emteknetnz) [reply en/08_Changelogs/5.2.0.md]: Updated

**Pull request merged**